app/controllers/outbound_webhooks_controller.rb

```diff
@@ -1,34 +1,28 @@
 # frozen_string_literal: true
 require 'samson/integration'
 
-class OutboundWebhooksController < ApplicationController
+class OutboundWebhooksController < ResourceController
   include CurrentProject
 
-  before_action :authorize_project_deployer!
+  prepend_before_action :authorize_project_deployer!
+  prepend_before_action :require_project
 
   def create
-    webhook = OutboundWebhook.new(outbound_webhook_params.merge(project_id: current_project.id))
-
-    if webhook.save
-      flash.delete(:error)
-      redirect_to project_webhooks_path(current_project)
-    else
-      flash[:error] = webhook.errors.full_messages.join(', ')
-      @new_outbound_webhook = webhook
-      render 'webhooks/index'
-    end
+    super(template: 'webhooks/index')
```

Comment:
FYI new pattern I'm starting since I'm getting sick of all these 1-off helper methods ... this seems to work fine and means we have less methods flying around